### PR TITLE
[ADD] account_invoice_line_complimentary

### PR DESCRIPTION
**Pull request by SirTakobi**

### CODE REVIEW

File: account_invoice_change_currency/models/account_change_currency.py
Comment (sergiocorato) [inline account_invoice_change_currency/models/account_change_currency.py:1]: This change is needed by the new module?

Comment (SirTakobi) [reply account_invoice_change_currency/models/account_change_currency.py:1]: Thanks for having a look!
Yes it is needed by the unit test contained in the new module.
Without this change, the unit test fails: you can see how in the test run of the previous commit (https://github.com/OCA/account-invoicing/actions/runs/4426084114/jobs/7761937518#step:8:236) and in the message of the commit containing this change.

**Review by sergiocorato — APPROVED** [review]
LGTM
Thanks!

File: account_invoice_line_complimentary/models/res_company.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+from odoo import fields, models
+
+
+class ResCompany (models.Model):
+    _inherit = 'res.company'
+
+    complimentary_account_id = fields.Many2one(
+        comodel_name='account.account',
+        string="Complimentary Account",
```
Comment (DucTruongKomit) [inline account_invoice_line_complimentary/models/res_company.py]: I think we don't need the string here

**Review by DucTruongKomit — COMMENTED** [review]
Hi! Here just 1 small comment!

File: account_invoice_line_complimentary/models/res_company.py (hunk 1 above)
Comment (SirTakobi) [reply account_invoice_line_complimentary/models/res_company.py]: :wave:
Thanks for having a look!
You are right, I have removed it.

**Review by mrcast — APPROVED** [review]
Functional test, ok!

**Review by eLBati — APPROVED** [review]
Thanks

**Review by DucTruongKomit — APPROVED** [review]
It should be okay now!

**Pull request merged**